src/lips.js

```diff
@@ -126,10 +126,10 @@ function log(x, regex = null) {
         }
         var value = global_env.get('repr')(x);
         if (regex === null || regex instanceof RegExp && regex.test(value)) {
-            console.log(global_env.get('type')(x) + ": " + value);
+            console.log(global_env.get('type')(x) + ": " + value); // @jcubic can you enter what this console.log returns here
         }
         if (literal) {
-            console.log(x);
+            console.log(x); // @jcubic can you enter what this console.log returns here
```

Comment:
If the second argument to the log function is true it will print whatever `repr` (representation) function returns (this function name is inspired by Python).